hack/verify-typecheck.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OPENSHIFT_ROOT=$(dirname "${BASH_SOURCE}")/..
```

Comment:
`$BASH_SOURCE` is an array, so

```sh
source "$(dirname "${BASH_SOURCE[0]}")/lib/init.sh"
```

It also does the `cd` for you, so omit that

Reply:
based on upstream, but done